python/ray/serve/_private/router.py

```diff
@@ -594,6 +599,17 @@ async def assign_request(
     ) -> ReplicaResult:
         """Assign a request to a replica and return the resulting object_ref."""
 
+        response_id = uuid.uuid4()
```

Comment:
Non-blocker: There is this [generate_request_id()](https://github.com/ray-project/ray/blob/671844571defbfc0c1349f3732e6233bf685283e/python/ray/serve/_private/utils.py#L552) maybe we can rename and reuse here?